metadata_gui_panel.py

```
@@ -173,6 +224,8 @@ def __init__(self, comic_data: GUIComicInfo) -> None:
         review_layout.addWidget(current_review)
         review_area.setWidget(review_container)
         review_area.setWidgetResizable(True)
+        review_panel = DashboardBox("Reviews", False)
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)